input/fsh/models.fsh

```diff
@@ -124,11 +124,9 @@ criteria are defined by FHIRPath expressions.
     with a Patient resource, a `forEachOrNull` on address will produce a row for each patient even if there are no addresses; it will
     simply set the address columns to `null`.
   """
-  * union 0..* contentReference http://hl7.org/fhir/uv/sql-on-fhir/StructureDefinition/ViewDefinition#ViewDefinition.select "TODO: Describe" """
-    TODO: Update this -- The result of each selection within the union will be combined according to the semantics of the 
-    union operator in FHIRPath. The results of the selected expressions must be of the same type, or 
-    able to be implicitly converted to a common type according to the FHIRPath data type conversion 
-    rules.
+  * unionAll 0..* contentReference http://hl7.org/fhir/uv/sql-on-fhir/StructureDefinition/ViewDefinition#ViewDefinition.select "Creates a union of all rows in the given select expressions." """
+    A `unionAll` combines the results of multiple `select` expressions. Each `select` under the `unionAll` must produce the same column names
```

Comment:
Hmm. I wonder if saying "`select`" here unintentionally might make people think that the "column"s inside a union branch don't count (because they are not behind a "select" property; rather, they're part of a "ViewDefinition.select" data structure). Not sure what to language to suggest, so maybe this just gets clarified through examples. 

Or, do we want to have a term like "Selection Structure" to describe the *type* of the backbone element called ViewDefinition.select ?

Reply:
Good point. I adopted the "selection structure" term as suggested.